src/plugins/ui/sort.ts

```diff
@@ -280,7 +280,7 @@ export class SortPlugin extends UIPlugin {
     const sortedIndex: number[] = sortedIndexOfSortTypeCells.map((x) => x.index);
 
     const [width, height]: [number, number] = [cells.length, cells[0].length];
-
+    const cellUpdates: any[] = [];
```

Comment:
`any` should be use very very rarely in Typescript :wink: It removes all type safety the language provides.
And most of the time, in the places where you don't (need to) know the type, `unknown` is preferred over `any`

```suggestion
    const cellUpdates: Omit<UpdateCellCommand, "type">[]= [];
```

Reply:
No problem, it's just that the variable `newCellValues` has `any` type and contains the `value` attribute which is not in `UpdateCellCommand`

Reply:
Done !